core/templates/dev/head/css/oppia.css

```diff
@@ -3440,6 +3440,9 @@ md-card.preview-conversation-skin-supplemental-card {
 .oppia-splash-section-two {
   background-color: #f2f2ee;
   color: #2c4841;
+  max-width: 1800px;
```

Comment:
Nit: we alphabetize css attr names, so max-width should come below margin-right.